tests/conformance/test_conformance.py

```diff
@@ -462,14 +641,22 @@ def blob_create_resumable_upload_session(client, _preconditions, **resources):
         blob_download_as_text,
         blobreader_read,
     ],
-    "storage.objects.list": [
-        client_list_blobs,
-        bucket_list_blobs,
-        bucket_delete,
-    ],  # S1 end
-    "storage.buckets.patch": [bucket_patch],  # S2/S3 start
+    "storage.objects.list": [client_list_blobs, bucket_list_blobs, bucket_delete],
+    "storage.serviceaccount.get": [client_get_service_account_email],  # S1 end
+    "storage.buckets.patch": [
+        bucket_patch,
+        bucket_make_public,
+        bucket_make_private,
+        bucket_acl_save,
```

Comment:
Ah, this is tricky-- I see that in our doc we define this as `storage.bucket_acl.patch` rather than a variant on `storage.buckets.patch`. The endpoint looks like this: https://cloud.google.com/storage/docs/json_api/v1/bucketAccessControls/patch .

Do we not have this separated out in the emulator routing?

Reply:
Thanks @tritone for the comments! We have this separated out in the emulator routing. It's the python client causing it to be tricky.

In python-storage, all of the ACL patch operations are done by calling `storage.buckets.patch` or `storage.objects.patch`, as the[ patch requests paths are either the bucket path or the object path](https://github.com/googleapis/python-storage/blob/3a489ad7e1cd2fe5adb443e781e72cf645a23331/google/cloud/storage/acl.py#L503). Therefore we're not calling any of the `storage.bucket_acl.patch` endpoints. Not sure if this is a bug in the library. I can check with other languages.

Reply:
Discussed offline, this is by design and not a problem.

Reply:
@tritone, @cojenco FWIW, my, rusty seven-year-old memory says that `storage.bucket_acl.patch`didn't used to work as desired.  See [this commit](https://github.com/googleapis/google-cloud-python/commit/b7f1b7747bb493a5df1982b6460a26023d6ab3c3) from this dim mists of time.